Homework #6

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script>
+System.out.println("hello worldd");
```

Comment:
This is syntax for Java, not JavaScript. In JavaScript, you would write `console.log("hello world);`.